### PR TITLE
Add instruction for adding launch image for iOS release

### PR DESCRIPTION
**Pull request by hellohuanlin**
Add instruction for adding launch image for iOS release

Issue: 

https://github.com/flutter/flutter/issues/97730

## Presubmit checklist
- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

**Review by InMatrix — APPROVED** [review]

File: src/deployment/ios.md [hunk 1]
```diff
@@ -140,6 +140,14 @@ app's icons:
 1. Verify the icon has been replaced by running your app using
    `flutter run`.
 
+## Add a launch image
+
+Similar to the app icon, you may also want to replace the placeholder launch image: 
+
+1. In the Xcode project navigator, select `Assets.xcassets` in the
+   `Runner` folder. Update the placeholder launch image with your own launch image. 
+1. Verify the launch image has been replaced by running your app using `flutter run`.
```
Comment (atsansone) [inline src/deployment/ios.md:149]: **suggestion:** Flip the sentence to remove passive voice.
```suggestion
1. Run  `flutter run` to verify that the launch image replacement.
```

**Review by atsansone — APPROVED** [review]
LGTM % one suggestion.

File: src/deployment/ios.md (hunk 1 above)
Comment (sfshaza2) [reply src/deployment/ios.md:149]: @atsansone, your suggestion was grammatically wrong. I've tweaked:

```suggestion
1. Verify the new launch image by hot restarting your app. (Don't use `hot reload`.)
```

**Review by sfshaza2 — APPROVED** [review]
LGTM with one comment.

**Comment by sfshaza2** [comment]
Thanks, @hellohuanlin!

**Pull request merged**